tests/playbooks/deb_remote.yaml

```diff
@@ -88,6 +88,75 @@
         that:
           - result.changed == false
 
+    - name: Modify sync_installer on remote
+      pulp.squeezer.deb_remote:
+        name: test_deb_remote
+        sync_installer: true
+        state: present
+      register: result
+    - name: Verify modify sync_installer
+      assert:
+        that:
+          - result.changed == true
+          - result.remote.sync_installer == true
```

Comment:
For the sake of brevity (and testing resources...) can you move all three options into the same two module invocations?

Reply:
@mdellweg, I combined the tests and added the fixture files. I've squashed the commits and added a `fixes #152` to the bottom of the commit messages. Let me know if there is anything out of order here.